news/929-bugfix.md

```diff
@@ -0,0 +1,7 @@
+Updated documentation
+Simplify SRAnonTool using external program semehr_anon.py
+Handle ConceptNameCodeSequence which has VR but no Value
+Ensure 'replaced' flag is not reset
+Write replacement DICOM whichever content tag is found
+Extract metadata from Mongo to go alongside anonymised text
+Redact numeric DICOM tags with all '9' not all 'X'
```

Comment:
unfortunately the github markdown parser will consider these to be one line, so when the changelog is generated they will be collapsed together. Could you add a heading and indent these points into a list?